docs/docusaurus/docs/core/define_expectations/define_expectations.md

```diff
@@ -20,6 +20,13 @@ import OverviewCard from '@site/src/components/OverviewCard';
     description="Create an instance of an Expectation"
     to="/core/define_expectations/create_an_expectation" 
     icon="/img/expectation_icon.svg" 
+  />
+    <LinkCard 
```

Comment:
**Note for reviewer** this landing page was indexing all pages in the section except for this one